### ISSUE TITLE
Probability of Continuing / Discount Modeling

### ISSUE THREAD

**Issue by juliusfrost** [juliusfrost/dreamer-pytorch#56]
Add the probability of continuing with a dense model. This is described very briefly in appendix A for use in Atari Environments. "We predict the discount factor from the latent state with a binary classiﬁer that is trained towards the soft labels of 0 and γ." 
`pcont` is the equivalent term in the TensorFlow implementation
